babel/src/board.js

```diff
@@ -247,14 +244,14 @@ export class Board {
           * 3. if the row isn't 0, it takes the orb from above
           * 4. if the row is 0, it creates a random orb
         */
-        _.each(_.range(this.height), x =>{
-            _.each(_.range(this.width), y => { //1
-                if (this.orbs[x][y] == 'X') {
-                    for (var z = x; z >= 0; z--) { //2
+        _.each(_.range(this.height), row =>{
+            _.each(_.range(this.width), col => { //1
+                if (this.orbs[row][col] == 'X') {
```

Comment:
Go with an empty string (or possibly a single space?). If someone is using an empty-ish string to represent an orb, they'll be having other problems outside of the code not working.

You can also look into printing some special characters (I prefer `'\u241E'`, aka the "Record Separator" special character (it looks like this printed: `'␞'`).